config/schema/entity_browser.schema.yml

```diff
@@ -36,7 +36,7 @@ entity_browser.browser.*:
           uuid:
             type: string
           label:
-            type: label
```

Comment:
This is reverting change that was introduced in 7d3c4441af607902dfbf78efae94cf577d72ba73. It is unrelated to this pull request.